{{cookiecutter.project_slug}}/setup.py

```diff
@@ -22,38 +22,38 @@
 from setuptools import setup, find_packages, convert_path
 
 # Requirements for {{cookiecutter.module_name}}
-requirements = ["OpenEye-orionplatform==0.3.2", "OpenEye-snowball==0.14.0"]
+requirements = ["OpenEye-orionplatform==1.0.0", "OpenEye-snowball==0.14.0"]
```

Comment:
Do we really want snowball in here as the default requirement? It can be a heavy weight requirement if you don't use it at all.

Reply:
if we want to keep snowball, itll have to be bumped to 0.16.0 for orionplatform 1.0.0 I beleive. But I agree that snowball is pretty enormous